index.html

```diff
@@ -1879,6 +1879,88 @@ <h3>Versioning Cryptography Suites</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Layer Cryptographic Options Appropriately</h3>
+
+        <p>
+Modern cryptographic algorithms provide a number of tunable parameters and
+options to ensure that the algorithms can meet different use case requirements.
+For example, embedded systems have limited processing and memory environments
+and might not have the resources to generate the strongest digital signatures
+for a given algorithm. Other environments, like financial trading systems,
+might only need to protect data for a day while the trade is occuring while
+other environments might need to protect data for multiple decades. To meet
+these needs, cryptographic algorithm designers often provide multiple ways to
+configure a cryptographic algorithm.
+        </p>
+        <p>
+Cryptographic library implementers often take the specifications created by
+cryptographic algorithm designers and specification authors and implement them
+such that all options are available to the application developers that use their
+libraries. This can be due to not knowing which combination of features a
+particular application developer might need for a given cryptographic algorithm.
+All options are often exposed to application developers.
+        </p>
+        <p>
+Application developers that use cryptographic libraries often do not have the
```

Comment:
I get the point you're making, but feel like some nuance is missed. A developers lack of mathematical understanding of [S-box construction](https://en.wikipedia.org/wiki/Rijndael_S-box) likely does not impact their ability to use AES affectively. Perhaps rephrasing "cryptographic algorithm designers" with "Security Practitioners" may work...

Reply:
> A developers lack of mathematical understanding of [S-box construction](https://en.wikipedia.org/wiki/Rijndael_S-box) likely does not impact their ability to use AES affectively.

True -- I think what we want to capture though are things like an application developer setting up their software to use AES-GCM, but because the chosen library also supports AES-CBC, an attacker can downgrade to it.

Reply:
My concern with "Security practitioners" is that it's too broad... that's like the union of "cryptographic algorithm designers, cryptographic library implementers, cryptographic suite designers, cryptographic suite implementers", right?

If we break the "groups of people that do things" into too many subgroups, it might be hard to follow the argument... if we combine too many of them into too few groups, it might not resonate with readers ("something seems off with these groups").

Got any concrete language suggestions, given the above?

Reply:
what about something like

```
Application developers that use cryptographic libraries often do not have the requisite cryptographic expertise and  knowledge necessary to appropriately select cryptographic parameters and options for a given application.
```

Reply:
Fixed in 3a788ad56421a496e0d6f0b8e91027ac8c3e9664.